arch/src/x86_64/cpuid_definitions/intel.rs

```diff
@@ -0,0 +1,1814 @@
+//! This module contains CPUID definitions for Intel CPUs. 
```

Comment:
I think I am missing some more general info how this file should be seen and handled.

Is this a complete list of all available cpuid leaves (and even subleaves) or are there missing pieces? Or is it only what we have decided is relevant for migration?

What would it mean for the profile generation and CHV CPUID handling if something is missing here? Will a VM then see the host value or is the leaf then not offered to the VM at all?

Would you recommend some policy if we have to change a value here later on? Is a change here later on backward compatible?